lib/smci_report.rb

```diff
@@ -182,10 +182,10 @@ def run
   private
 
   def header_row
-    %w(title author_list pmid wos_uid sul_pub_id doi doi_url publisher journal volume articlenumber supplement issue pages mesh pub_year pub_date provenance profiles_author_last_name profiles_author_first_name profiles_author_sunet profiles_author_cap_profile_id profiles_author_employee_id profiles_author_email pub_harvested_date apa_citation mla_citation chicago_citation)
+    %w(title author_list pmid wos_uid sul_pub_id doi doi_url publisher journal volume articlenumber supplement issue pages mesh pub_year pub_date provenance profiles_author_last_name profiles_author_first_name profiles_author_sunet profiles_author_cap_profile_id profiles_author_employee_id profiles_author_email publication_status pub_harvested_date apa_citation mla_citation chicago_citation)
   end
 
-  def output_row(pub_hash, author, harvested_at)
+  def output_row(pub_hash, author, harvested_at, publication_status)
```

Comment:
Can you use kwargs for this new field? That makes it easier to understand what we're passing in.

Reply:
Yes, that makes it easier to understand.  I also made several defaults for one spot where it is called that made it cleaned too.